sig-cluster-lifecycle/README.md

```diff
@@ -116,7 +116,7 @@ The following [subprojects][subproject-definition] are owned by sig-cluster-life
 - **Contact:**
   - Slack: [#cluster-api-vsphere](https://kubernetes.slack.com/messages/cluster-api-vsphere)
 - **Meetings:**
-  - Cluster API vSphere meeting: [Tuesdays at 11:00 PT (Pacific Time)](https://zoom.us/j/875399243) (biweekly starting Wednesday March 31st, 2020). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=PT%20%28Pacific%20Time%29).
+  - Cluster API vSphere meeting: [Thursdays at 10:00 PT (Pacific Time)](https://zoom.us/j/875399243) (biweekly starting Thursday June 24th, 2020). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=10:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
June 25th is a Thursday

Reply:
wow, good catch! Happy to LGTM or approve once this is fixed. @yastij

Reply:
whoops, good catch, updated.